forge-gui/res/cardsfolder/upcoming/ghastly_death_tyrant.txt

```diff
@@ -0,0 +1,11 @@
+Name:Ghastly Death Tyrant
+ManaCost:4 B B
+Types:Creature Beholder Skeleton
+PT:6/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCharm | TriggerDescription$ When CARDNAME enters the battlefield, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DB$Destroy,DBPumpAll
+SVar:DB$Destroy:DB$ Destroy | ValidTgts$ Enchantment | TgtPrompt$ Select target enchantment | SubAbility$ DBLoseLife |SpellDescription$ Disintegration Ray — Destroy target enchantment an opponent controls. You lose life equal to its mana value.
```

Comment:
PrecostDesc$ Disintegration Ray — | SpellDescription$ Destroy target enchantment an opponent controls. You lose life equal to its mana value.